android/src/main/java/com/reactnativeescposprinter/EscPosPrinterModule.java

```diff
@@ -498,11 +497,11 @@ public void stopMonitorPrinter(Promise promise) {
   }
 
   @ReactMethod
-  public void printBuffer(ReadableArray printBuffer, Promise promise) {
+  public void printBuffer(ReadableArray printBuffer, int timeout, Promise promise) {
```

Comment:
I think it'll be better if we could pass an object here like params in discovery. And I suppose it should be optional.

Reply:
will do